### PR TITLE
Use cloudflare origin cert for managed instances

### PR DESCRIPTION
**Pull request by michaellzc**
Context: https://sourcegraph.slack.com/archives/C02Q9G9A59S/p1650475567847399

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md [hunk 1]
```diff
@@ -9,12 +9,16 @@ For basic operations like accessing an instance for these steps, see [managed in
...
-   - `export TF_VAR_opsgenie_webhook=<OpsGenie Webhook value>`
-     - This can be found in the [Managed Instances vault](https://my.1password.com/vaults/nwbckdjmg4p7y4ntestrtopkuu/allitems/d64bhllfw4wyybqnd4c3wvca2m)
+
+- `export VERSION=vMAJOR.MINOR.PATH`
+- `export COMPANY=$COMPANY`
+- `export PROJECT_PREFIX=sourcegraph-managed` (should match GCP project prefix)
+- `export PROJECT_ID=$PROJECT_PREFIX-$COMPANY`
+- `export TF_VAR_opsgenie_webhook=<OpsGenie Webhook value>`
+  - This can be found in the [Managed Instances vault](https://my.1password.com/vaults/nwbckdjmg4p7y4ntestrtopkuu/allitems/d64bhllfw4wyybqnd4c3wvca2m)
+- `export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)`
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md]:
```suggestion
- `export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)`
```

File: content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md [hunk 2]
```diff
@@ -9,12 +9,16 @@ For basic operations like accessing an instance for these steps, see [managed in
...
+- `export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)`
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md]:
```suggestion
- `export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)`
```

**Review by daxmc99 — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md (hunk 1 above)
Comment (michaellzc) [reply content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md]: TIL you can use `latest` 

nice!

File: content/departments/product-engineering/engineering/cloud/devops/managed/resume_process.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ export CURRENT_DEPLOYMENT=$(gcloud compute snapshots list --format json --sort-b
 # The value can be found in the Managed Instances vault
 # https://my.1password.com/vaults/nwbckdjmg4p7y4ntestrtopkuu/allitems/d64bhllfw4wyybqnd4c3wvca2m
 export TF_VAR_opsgenie_webhook=<OpsGenie Webhook value>
+
+export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
+export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/resume_process.md]:
```suggestion
export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```

File: content/departments/product-engineering/engineering/cloud/devops/managed/suspend_process.md [hunk 4]
```diff
@@ -47,6 +47,9 @@ export CURRENT_DEPLOYMENT=$(gcloud compute instances list --project=${PROJECT_PR
 # The value can be found in the Managed Instances vault
 # https://my.1password.com/vaults/nwbckdjmg4p7y4ntestrtopkuu/allitems/d64bhllfw4wyybqnd4c3wvca2m
 export TF_VAR_opsgenie_webhook=<OpsGenie Webhook value>
+
+export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
+export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/suspend_process.md]:
```suggestion
export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```

File: content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md [hunk 5]
```diff
@@ -65,6 +65,9 @@ export TF_VAR_opsgenie_webhook=<OpsGenie Webhook value>
 export OLD_DEPLOYMENT=$(gcloud compute instances list --project "$PROJECT_PREFIX-$CUSTOMER" | grep -v "executors" | awk 'NR>1 { if ($1 ~ "-red-") print "red"; else print "black"; }')
 # the instance we will create
 export NEW_DEPLOYMENT=$([ "$OLD_DEPLOYMENT" = "red" ] && echo "black" || echo "red")
+# cf origin cert
+export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
+export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access 1 --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md]:
```suggestion
export TF_VAR_cf_origin_cert_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_CERT" | base64)
export TF_VAR_cf_origin_private_key_base64=$(gcloud secrets versions access latest --project=sourcegraph-dev --secret="SOURCEGRAPH_WILDCARD_KEY" | base64)
```

**Pull request merged**